LICENSE.txt

```diff
@@ -186,8 +186,6 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2020 Real Folk Inc.
```

Comment:
We completely agree the copyright line should not have been removed. Should we revert this to have the actual copyright owner of Province of British Columbia?

Reply:
My recommendation is to leave the copyright statement as is `Copyright 2020 Real Folk Inc.` since removing it or modifying it would be, in my non-legal opinion, a violation of the license it has been released under. The copyright statement was created two years ago, so presumably intellectual property (IP) was with the vendor then and absent any good reason, should remain with the vendor now. Any subsequent contributions to the code base, may take the form of adding an _additional_ copyright statement with the appropriate year distinguishing what changes the copyright applies to. My understanding is that if gov employees were the sole creators of content, then IP would be with the gov, otherwise it is likely with the vendor. Contractual language may resemble the following: "All intellectual property rights, including copyright, in the Contributions that are not owned by third parties will be owned by the Contractor." Further info is here: https://developer.gov.bc.ca/Licensing-Your-Project 
Inviting @SteveChapmanBCDX to weigh in.